src/grammar.lalrpop

```diff
@@ -561,6 +562,45 @@ ArgNames: ArgNames = {
     <label:VarName> <name:DiscardName> => ArgNames::LabelledDiscard { label, name },
 }
 
+RecordUpdate: UntypedExpr = {
+    // <s:@L> <m:VarName> "#" <n:UpName> <args:RecordUpdateArgs> <e:@L> => UntypedExpr::RecordUpdate {
```

Comment:
Right now I don't support updating records that were imported from another module. In theory this production rule should allow me to handle them, but it makes the grammar ambiguous. We should be able to rearrange the grammar to make things work, but I'm not sure what the best approach is for doing that.

Do you have any ideas?

Reply:
I tried a bit of inlining trickery to see if I could get this working but unfortunately got nowhere. Not very helpful, sorry!

Reply:
Something like this perhaps? https://github.com/gleam-lang/gleam/commit/2d63872cc988b9e007638cbde495fe93a5b91535

Reply:
I got it working with a variant of this, thank you!